content/docs/studio/create-view.md

```diff
@@ -1,67 +1,71 @@
+   listed.
+
+> To create views from your GitHub repositories, you must install the DVC Studio
+> GitHub app. Refer to the section on
+> [GitHub app installation](#dvc-studio-github-app-installation) for more
+> details.
 
 3. Open the organization whose repository you want to connect to. You can also
-   use the search bar to search directly for the repo which you want to connect
-   to. ![](https://static.iterative.ai/img/studio/select_repo.png)
+   use the search bar to search directly for the repository. Click on the
+   required repository to connect to it.
 
-4. Select the Git repository for which you want to create a view. For this,
-   hover over the required repository and click `Connect`.
+   ![](https://static.iterative.ai/img/studio/select_repo_v2.png)
 
-5. Once you click on `Connect`, you will be able to specify advanced settings
-   for the connection.
-   ![](https://static.iterative.ai/img/studio/view_settings.png)
+4. You will be able to specify additional settings for the connection.
 
-> In most scenarios, you can skip the advanced settings. Refer to the
-> [Advanced Settings](#advanced-settings) section below for more details.
+   ![](https://static.iterative.ai/img/studio/view_settings_v2.png)
 
-You should now see that a view has been added in your dashboard.
+> In most scenarios, you can skip these additional View settings. Refer to the
+> [View Settings](/doc/studio/view-settings) section for more details.
 
-![](https://static.iterative.ai/img/studio/view_added.png)
+You should now see that a view has been added in your dashboard.
 
 If your project requires any of the advanced settings, then remember to
 configure them by opening the View settings. Otherwise, your view may not work
 as expected. To go to View settings, click on the
-![](https://static.iterative.ai/img/studio/view_open_settings_icon.png) icon in
-the view. In the menu that opens up (highlighted in yellow in the screenshot
-below), click on `Settings`.
+![](https://static.iterative.ai/img/studio/view_open_settings_icon_v2.png) icon
+in the view. In the menu that opens up, click on `Settings`.
 
-![](https://static.iterative.ai/img/studio/view_open_settings.png)
+![](https://static.iterative.ai/img/studio/view_open_settings_v2.png)
 
-## Advanced Settings
+## Share a view
 
-You will need to specify advanced settings in the following scenarios.
+You can share your views on the web. Simply click on the button labelled
+`Private` next to the name of the view. In the menu that pops up, turn on
+`Share to Web`.
 
-> These settings can be skipped when creating the view -- they can be edited
-> later.
+![](https://static.iterative.ai/img/studio/view_share_v2.png)
 
-- **Monorepo:** If you have connected to a
-  [monorepo](https://en.wikipedia.org/wiki/Monorepo), then specify the full path
-  to the root directory of the project that you want to connect.
+You can change a shared (public) view and make it private by turning off
+`Share to web`.
```

Comment:
```suggestion
`Share to Web`.
```

Reply:
btw is "share to web" a better label than simply "share?"

"to web" implies there are other options ("by email?")

Reply:
`Share to web` is how the link appears on Studio. So, not changing it here. I think `Publish to web` may be a better option. But it needs to be changed in Studio first, before changing in the docs.

Reply:
yep, create a ticket on in the `iterative/viewer` please. Good cath @casperdcl .

Reply:
Created https://github.com/iterative/viewer/issues/2098